source/simulator/core.cc

```diff
@@ -2015,11 +2015,15 @@ namespace aspect
                                         parameters.initial_adaptive_refinement;
     pre_refinement_step = 0;
 
+    // shall we start at id=0 when not resuming??
+
```

Comment:
yes, but you do that anyway below. so this comment can go?

We should probably initialize this to `numbers::invalid_unsigned_int` in the constructor?